Cargo.toml

```diff
@@ -1,8 +1,6 @@
 [package]
-name = "crate-template"
+name = "vm-fdt"
 version = "0.1.0"
-authors = [TODO]
+authors = ["The Chromium OS Authors"]
 license = "Apache-2.0 OR BSD-3-Clause"
```

Comment:
All files are licensed under BSD-3-Clause, should we remove "Apache-2.0"? Or change source to " Apache-2.0 OR BSD-3-Clause"? 
I remember there was a discussion about preferring "Apache-2.0 OR BSD-3-Clause".

Reply:
I can license them under "Apache-2.0 OR BSD-3-Clause" if that is preferred - updated to use the SPDX file header (please sanity check that this looks correct).